lib/app_text/second_page_text.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class AppText {
```

Comment:
zmień nazwy tych tekstów na coś w stylu text, textSmall, textBold, title, header etc

Reply:
jak potrzebujesz coś w innym kolorze niż domyślny(czyli strzelam, że czarny) to robisz w ekranie AppText.textSmall.copyWith(color: color)